README.md

```diff
@@ -3,11 +3,49 @@
 [![Average time to resolve an issue](http://isitmaintained.com/badge/resolution/azure/bicep-lz-vending.svg)](http://isitmaintained.com/project/azure/bicep-lz-vending "Average time to resolve an issue")
 [![Percentage of issues still open](http://isitmaintained.com/badge/open/azure/bicep-lz-vending.svg)](http://isitmaintained.com/project/azure/bicep-lz-vending "Percentage of issues still open")
 
+_____________________________________________________________________________________________________________________________________
+
+## ⚠️⚠️ Repository archival notice ⚠️⚠️
+
+We would like to inform you that this repository has been **archived** and the Bicep landing zone vending module for Azure now has a new home in the [Bicep public registry](https://aka.ms/lz-vending/bicep).
+
+### What does this mean?
+
+- This repository is now in a **read-only** state.
+- No further issues, pull requests, or updates will be maintained on this repository.
+- All future releases, fixes and improvements will be managed in the new module hosted on the [Bicep public registry](https://aka.ms/lz-vending/bicep).
+- We will continue to leverage the [Wiki](https://github.com/azure/bicep-lz-vending/wiki) in this repository as the module's documentation and guidance location.
+
+### Why was this done?
+
+We are converting this module to be an [Azure Verified pattern module](https://aka.ms/avm) to better align with the [Azure Verified Modules](https://azure.github.io/Azure-Verified-Modules/concepts/what-why-how/) initiative. This will help us align with the Well-architected framework guidance, have improved [module support](https://azure.github.io/Azure-Verified-Modules/help-support/module-support/) and provide you with a consistent experience consuming [AVM Bicep modules](https://azure.github.io/Azure-Verified-Modules/indexes/bicep/).
+
+📒 If you are not familiar with [Azure Verified Modules](https://aka.ms/avm), you can watch the following Youtube videos for a quick overview:
+
+- [An Introduction to Azure Verified Modules (AVM) by the Customer Architecture & Engineering team](https://youtu.be/JbIMrJKW5N0?si=4v69hlyBbINufHEO) 📽️🎬🎞️
+- [Azure Verified Modules Overview by John Savill](https://youtu.be/3FeIFHaJOtg?si=ZDXJ3EPwT9Xlzq2P) 📽️🎬🎞️
+
+### If I'm already using the Bicep Subscription vending module from the Bicep public registry, what do I need to change?
```

Comment:
lets add something here about `v1.5.2` becoming `v0.1.0` in AVM, or whatever the next release number would have been 👍